packages/create-plugin/src/constants.ts

```diff
@@ -99,6 +99,10 @@ export const TEXT = {
   updateNpmDependenciesSuccess: 'Successfully updated the NPM dependencies.',
   updateNpmDependenciesAborted: 'No NPM dependencies have been updated.',
 
+  addProvisioning: '**Would you like to add provisioning files?**',
```

Comment:
```suggestion
  addProvisioning: '**Do you want to add provisioning files?**',
```

Reply:
Doc style prefers not to use conditional verb tenses. https://developers.google.com/style/tense